supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionCommands.scala

```diff
@@ -95,6 +95,6 @@ object ActionCommands {
     val withProject = s"-u ${path.projectId}"
 
     s"""${f(withoutProject)} 2> gsutil_output.txt; RC_GSUTIL=$$?; if [[ "$$RC_GSUTIL" -eq 1 ]]; then
-       | grep "$BucketIsRequesterPaysErrorMessage" gsutil_output.txt && echo "Retrying with user project"; ${f(withProject)}; fi """.stripMargin
+       | grep "$BucketIsRequesterPaysErrorMessage" gsutil_output.txt && echo "Retrying with user project" && ${f(withProject)}; fi """.stripMargin
```

Comment:
1. maybe, not sure it matters
2. not sure there's another way of testing for variables and chaining commands with nested ifs would be 🤢 
3. apparently not, this is running under `/bin/sh` 😛

Reply:
1. Given that we're `echo`ing, I have to assume that "what ends up on stdout" is important here
2. Since it *is* a logical nested `if` - even if it's split over different control flow styles - perhaps the `grep` be moved into the if condition?
3. Great!